### ISSUE TITLE
Bad detection from a pre-trainded model 

### ISSUE THREAD

**Issue by jadhm** [tensorflow/models#5376]
OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Linux Ubuntu 16.04
TensorFlow installed from (source or binary):
TensorFlow version (use command below):('v1.9.0-0-g25c197e023', '1.9.0')
GPU model and memory:Quadro M1000M , 4 Gb
Bazel version: NA
CUDA/cuDNN version:NA
Exact command to reproduce: NA
Mobile device : NA
----------------------------------------------------------------------------

Hi,

For detecting a particular machine for my assignment I used the  pre-trained model :**faster_rcnn_inception_v2_coco_2018_01_28** . After I finished training I tested it. The good thing is it detects and labels the machine very well but the bad thing is, it sees me as human sometimes as that machine, also some other stuffs like chair or dog, it seems like it had old memories and it is still confused. So, i am wondering did i do something wrong ? and what could be the problem, knowing that i only have one class and i used around 3500 images of that machine and the image is 640 * 480. The configurations that i used for training are the following : 
Can anyone please help !

>  Faster R-CNN with Inception v2, configuration for MSCOCO Dataset.
>  Users should configure the fine_tune_checkpoint field in the train config as
>  well as the label_map_path and input_path fields in the train_input_reader and
>  eval_input_reader. Search for "PATH_TO_BE_CONFIGURED" to find the fields that
> should be configured 
> 
> 
> model {
>   faster_rcnn {
>     num_classes: 1
>     image_resizer {
>       keep_aspect_ratio_resizer {
>         min_dimension: 600
>         max_dimension: 1024
>       }
>     }
>     feature_extractor {
>       type: 'faster_rcnn_inception_v2'
>       first_stage_features_stride: 16
>     }
>     first_stage_anchor_generator {
>       grid_anchor_generator {
>         scales: [0.25, 0.5, 1.0, 2.0]
>         aspect_ratios: [0.5, 1.0, 2.0]
>         height_stride: 16
>         width_stride: 16
>       }
>     }
>     first_stage_box_predictor_conv_hyperparams {
>       op: CONV
>       regularizer {
>         l2_regularizer {
>           weight: 0.0
>         }
>       }
>       initializer {
>         truncated_normal_initializer {
>           stddev: 0.01
>         }
>       }
>     }
>     first_stage_nms_score_threshold: 0.0
>     first_stage_nms_iou_threshold: 0.7
>     first_stage_max_proposals: 300
>     first_stage_localization_loss_weight: 2.0
>     first_stage_objectness_loss_weight: 1.0
>     initial_crop_size: 14
>     maxpool_kernel_size: 2
>     maxpool_stride: 2
>     second_stage_box_predictor {
>       mask_rcnn_box_predictor {
>         use_dropout: false
>         dropout_keep_probability: 1.0
>         fc_hyperparams {
>           op: FC
>           regularizer {
>             l2_regularizer {
>               weight: 0.0
>             }
>           }
>           initializer {
>             variance_scaling_initializer {
>               factor: 1.0
>               uniform: true
>               mode: FAN_AVG
>             }
>           }
>         }
>       }
>     }
>     second_stage_post_processing {
>       batch_non_max_suppression {
>         score_threshold: 0.0
>         iou_threshold: 0.6
>         max_detections_per_class: 100
>         max_total_detections: 300
>       }
>       score_converter: SOFTMAX
>     }
>     second_stage_localization_loss_weight: 2.0
>     second_stage_classification_loss_weight: 1.0
>   }
> }
> 
> train_config: {
>   batch_size: 4
>   optimizer {
>     momentum_optimizer: {
>       learning_rate: {
>         manual_step_learning_rate {
>           initial_learning_rate: 0.0002
>           schedule {
>             step: 900000
>             learning_rate: .00002
>           }
>           schedule {
>             step: 1200000
>             learning_rate: .000002
>           }
>         }
>       }
>       momentum_optimizer_value: 0.9
>     }
>     use_moving_average: false
>   }
>   gradient_clipping_by_norm: 10.0
>   fine_tune_checkpoint: "object_detection/faster_rcnn_inception_v2_coco_2018_01_28/model.ckpt"
>   from_detection_checkpoint: true
>    Note: The below line limits the training process to 200K steps, which we
>   empirically found to be sufficient enough to train the COCO dataset. This
>    effectively bypasses the learning rate schedule (the learning rate will
>    never decay). Remove the below line to train indefinitely.
>   num_steps: 200000
>   data_augmentation_options {
>     random_horizontal_flip {
>     }
>   }
> }
> 
> train_input_reader: {
>   tf_record_input_reader {
>     input_path: "object_detection/data/train.record"
>   }
>   label_map_path: "object_detection/training/object-detection.pbtxt"
> }
> 
> eval_config: {
>   num_examples: 2858
>    Note: The below line limits the evaluation process to 10 evaluations.
>    Remove the below line to evaluate indefinitely.
>   max_evals: 10
> }
> 
> eval_input_reader: {
>   tf_record_input_reader {
>     input_path: "object_detection/data/test.record"
>   }
>   label_map_path: "object_detection/training/object-detection.pbtxt"
>   shuffle: false
>   num_readers: 1
> }

**Comment by harshini-gadige** [comment]
This question is better asked on [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
